packages/tools/src/tools/annotation/RectangleROITool.ts

```diff
@@ -968,6 +968,14 @@ class RectangleROITool extends AnnotationTool {
   };
 
   _isInsideVolume = (index1, index2, dimensions) => {
+    // for wsiViewport
+    if (index1[1] < 0) {
```

Comment:
This doesn't look good. Could you please check if the viewport type is WSI before proceeding? I imagine this might disrupt other types of viewports.